Frameworks/Foundation/NSMutableArray.mm

```diff
@@ -557,11 +557,23 @@ - (void)replaceObjectsInRange:(NSRange)aRange withObjectsFromArray:(NSArray*)oth
 }
 
 /**
- @Status Caveat
- @Notes Options are currently ignored
+ @Status Interoperable
+ @Notes NSSortConcurrent is ignored, but this is compatible with reference documentation
 */
 - (void)sortWithOptions:(NSSortOptions)opts usingComparator:(NSComparator)cmptr {
-    [self sortUsingFunction:CFNSBlockCompare context:cmptr];
+    if (NSSortStable) {
```

Comment:
Oh no! good catch.

---
In reply to: [72299337](https://github.com/Microsoft/WinObjC/pull/673#discussion_r72299337) [](ancestors = 72299337)